src/box/sql/build.c

```diff
@@ -2337,10 +2337,17 @@ sql_drop_constraint(struct Parse *parse_context)
 		parse_context->is_aborted = true;
 		return;
 	}
+	struct Vdbe *v = sqlGetVdbe(parse_context);
+	assert(v != NULL);
 	struct constraint_id *id = space_find_constraint_id(space, name);
 	if (id == NULL) {
-		diag_set(ClientError, ER_NO_SUCH_CONSTRAINT, name, table_name);
-		parse_context->is_aborted = true;
+		char *name_copy = sqlDbStrDup(parse_context->db, name);
+		if (name_copy == NULL) {
+			parse_context->is_aborted = true;
+			return;
+		}
+		sqlVdbeAddOp4(v, OP_DropConstraint, space->def->id, 0, 0,
```

Comment:
I suggest to be more specific and call this opcode `OP_DropTupleConstraint` or something. Unless you plan to make it work with all constraints in the future - with unique indexes, fk, ck.

Reply:
I renamed this opcode to `OP_DropTupleConstraint`.